packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -94,248 +101,111 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/REA
+     {{- end }}
+     imagePullPolicy: {{ .Values.sidecar.imagePullPolicy }}
+     env:
+@@ -108,9 +107,9 @@
+ {{- if .Values.sidecar.notifiers.enabled }}
+   - name: {{ template "grafana.name" . }}-sc-notifiers
+     {{- if .Values.sidecar.image.sha }}
+-    image: "{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}@sha256:{{ .Values.sidecar.image.sha }}"
++    image: "{{ template "system_default_registry" . }}{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}@sha256:{{ .Values.sidecar.image.sha }}"
+     {{- else }}
 -    image: "{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}"
 +    image: "{{ template "system_default_registry" . }}{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}"
+     {{- end }}
      imagePullPolicy: {{ .Values.sidecar.imagePullPolicy }}
      env:
-       - name: METHOD
-@@ -96,7 +96,7 @@
- containers:
+@@ -153,9 +152,9 @@
  {{- if .Values.sidecar.dashboards.enabled }}
    - name: {{ template "grafana.name" . }}-sc-dashboard
+     {{- if .Values.sidecar.image.sha }}
+-    image: "{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}@sha256:{{ .Values.sidecar.image.sha }}"
++    image: "{{ template "system_default_registry" . }}{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}@sha256:{{ .Values.sidecar.image.sha }}"
+     {{- else }}
 -    image: "{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}"
 +    image: "{{ template "system_default_registry" . }}{{ .Values.sidecar.image.repository }}:{{ .Values.sidecar.image.tag }}"
+     {{- end }}
      imagePullPolicy: {{ .Values.sidecar.imagePullPolicy }}
      env:
-       - name: METHOD
-@@ -122,7 +122,7 @@
-         mountPath: {{ .Values.sidecar.dashboards.folder | quote }}
+@@ -187,9 +186,9 @@
  {{- end}}
    - name: {{ .Chart.Name }}
+     {{- if .Values.image.sha }}
+-    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}@sha256:{{ .Values.image.sha }}"
++    image: "{{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}@sha256:{{ .Values.image.sha }}"
+     {{- else }}
 -    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
 +    image: "{{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+     {{- end }}
      imagePullPolicy: {{ .Values.image.pullPolicy }}
    {{- if .Values.command }}
-     command:
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/charts/grafana/values.yaml packages/rancher-monitoring/charts/charts/grafana/values.yaml
 --- packages/rancher-monitoring/charts-original/charts/grafana/values.yaml
 +++ packages/rancher-monitoring/charts/charts/grafana/values.yaml
-@@ -49,8 +49,8 @@
+@@ -49,7 +49,7 @@
  # schedulerName: "default-scheduler"
  
  image:
 -  repository: grafana/grafana
--  tag: 7.0.5
 +  repository: rancher/grafana-grafana
-+  tag: 7.0.6
+   tag: 7.1.5
```

Comment:
assuming these new tags have been mirrored already

Reply:
Being tracked in https://github.com/rancher/image-mirror/pull/35